includes/Story_Post_Type.php

```diff
@@ -614,4 +620,33 @@ public function add_to_jetpack_sitemap( $post_types ) {
 
 		return $post_types;
 	}
+
+	/**
+	 * Change the content to an embedded player
+	 *
+	 * @param string $content Current content of filter.
+	 *
+	 * @return string
+	 */
+	public function embed_player( $content ) {
+		$post = get_post();
+
+		if ( is_feed() ) {
+			return $content;
+		}
+
+		if ( ! is_search() && ! is_post_type_archive() ) {
+			return $content;
+		}
+
+		if ( $post instanceof WP_Post && self::POST_TYPE_SLUG === $post->post_type ) {
+			$story = new Story();
+			$story->load_from_post( $post );
+
+			$embed   = new Embed( $story );
+			$content = $embed->render();
+		}
+
+		return $content;
```

Comment:
Nit, but could do bail early for consistency:

```suggestion
		if ( ! $post instanceof WP_Post || self::POST_TYPE_SLUG !== $post->post_type ) {
			return $content;
		}

		$story = new Story();
		$story->load_from_post( $post );

		$embed   = new Embed( $story );
		return $embed->render();
```

Reply:
You bail early to start a large amount of processing this only stop 3 lines. It also means that I would have to write a test for another post to check this work. I don't see the point.

Reply:
Single return functions are much cleaner.

Reply:
The branch coverage would be the same, so I don't see how this would require another test, but alas 🤷